engine/class_modules/sc_druid.cpp

```diff
@@ -8579,15 +8570,14 @@ struct convoke_the_spirits_t : public druid_spell_t
                    };
 
     cast_list.insert( cast_list.end(),
-                      static_cast<size_t>( rng().range( guidance ? 2.5 : 4, guidance ? 7.5 : 9 ) ),
+                      rng().range<size_t>( guidance ? 2.5 : 4, guidance ? 7.5 : 9 ),
                       CAST_OFFSPEC );
 
     insert_exceptional( CAST_FERAL_FRENZY );
 
     cast_list.insert(
         cast_list.end(),
-        _clamp_and_cast( rng().gauss( guidance ? 3 : 4.2, guidance ? 0.8 : 0.9360890055, true ), 0,
-                         max_ticks - cast_list.size() ),
+        rng().gauss_ab<size_t>( guidance ? 3 : 4.2, guidance ? 0.8 : 0.9360890055, 0, max_ticks - cast_list.size() ),
```

Comment:
See above, this should use the double version and cast to size_t after returning.